QRCoder/QRCodeGenerator/Polynom.cs

```diff
@@ -258,19 +258,14 @@ void ThrowArgumentOutOfRangeException()
             private static void ReturnArray(PolynomItem[] array)
             {
                 if (array == null)
-                    ThrowArgumentNullException();
```

Comment:
Dispose calls this private method and Dispose should not throw when called twice